Realm/RLMArray.h

```diff
@@ -26,59 +26,59 @@ NS_ASSUME_NONNULL_BEGIN
 
 /**
 
- RLMArray is the container type in Realm used to define to-many relationships.
+ `RLMArray` is the container type in Realm used to define to-many relationships.
 
- Unlike an NSArray, RLMArrays hold a single type, specified by the `objectClassName` property.
+ Unlike an `NSArray`, `RLMArray`s hold a single type, specified by the `objectClassName` property.
  This is referred to in these docs as the “type” of the array.
 
- When declaring an RLMArray property, the type must be marked as conforming to a
+ When declaring an `RLMArray` property, the type must be marked as conforming to a
  protocol by the same name as the objects it should contain (see the
- `RLM_ARRAY_TYPE` macro). RLMArray properties can also use Objective-C generics
- if available. For example:
+ `RLM_ARRAY_TYPE` macro). In addition, the property can be made generic for better
```

Comment:
"made generic" doesn't feel like correct phrasing to me. The _type_ is the thing that can be made generic. When declaring the property you're only choosing whether to opt into to using generics. Something partway between the original and your version seems like a better fit:
> the property can be declared using Objective-C generics for better compile-time type safety